src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,19 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
-     * Imagine that some user wants to login to your site.
+     * Imagine that some user wants to log in to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
      * @param email - user's email
-     * @param password - user's password
+     * @param inputtedPassword - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
-    public boolean login(String email, String password) {
-        return false;
+    public boolean login(String email, String inputtedPassword) {
+        User userByEmail = new UserService().findByEmail(email);
+        return userByEmail.getPassword().equals(inputtedPassword);
```

Comment:
What happens when an email arrives that is not in the users' list? Also think about `new UserService()` on every call `login` method, is there a good solution?